include/bark/feature_detection.hpp

```diff
@@ -49,4 +57,10 @@
 #    define BARK_CONSTEXPR
 #endif
 
+#if ASIO_HAS_LOCAL_SOCKETS && BARK_MSC_VERSION == 0
```

Comment:
I would be thinking that ASIO_HAS_LOCAL_SOCKETS, should be enough?

Reply:
I have tried to remove it now - I don't remember if I had trouble with it back then.

Reply:
I get a test error because it tries to run some UDS code on Windows when not having both 🤔 

```
D:\a\bark\bark\test\source\transports\async_uds_transport_tests.cpp(21):
TEST SUITE: AsyncUDSTransport
TEST CASE:  when sending a message over UDS, the server should receive it
D:\a\bark\bark\test\source\transports\async_uds_transport_tests.cpp(21): FATAL ERROR: test case CRASHED: SIGABRT - Abort (abnormal termination) signal
```